### DISCUSSION TITLE
macOS app does not work

### DISCUSSION THREAD

**Discussion by juliosong** [xiaoyifang/goldendict-ng#962] [General]
This is such a nice project. However, it seems neither of the two dmg installation packages works on the latest macOS system (Ventura 13.4.1). Is there going to be a new version? Thanks!

**Comment by shenlebantongying** [comment]
You can download pre-release builds https://github.com/xiaoyifang/goldendict-ng/releases

**Reply by juliosong** [reply]
Thanks for the swift reply! I have already downloaded a Mac-compatible version just now from the website but might check this one out again in the future.

**Comment by xiaoyifang** [comment]
> This is such a nice project. However, it seems neither of the two dmg installation packages works on the latest macOS system (Ventura 13.4.1). Is there going to be a new version? Thanks!

What's the error message? need more information.

**Reply by juliosong** [reply]
Hi! I have just tried again and the problem persists. Here's the error message:
<img width="255" alt="Screenshot 2023-07-14 at 21 45 18" src="https://github.com/xiaoyifang/goldendict-ng/assets/40522993/734fd59d-8bf7-4237-b3dd-565069803969">

I clicked OK and went to Privacy & Security in System Settings, where I saw the message below:
<img width="471" alt="Screenshot 2023-07-14 at 21 45 26" src="https://github.com/xiaoyifang/goldendict-ng/assets/40522993/e07046cc-fc76-4b0c-800a-2c406a60fe56">

Then I clicked Open Anyway and saw the message below:
<img width="254" alt="Screenshot 2023-07-14 at 21 45 40" src="https://github.com/xiaoyifang/goldendict-ng/assets/40522993/64bf6a1e-12b6-4729-baba-c97db938f58f">

I clicked Open and saw the following window:
<img width="980" alt="Screenshot 2023-07-14 at 21 45 53" src="https://github.com/xiaoyifang/goldendict-ng/assets/40522993/c73b1c0f-8d47-468a-93d3-c8d953398a1e">

I have copied and pasted the crash report into a text file in case that is useful:
[GoldenDict crash report.txt](https://github.com/xiaoyifang/goldendict-ng/files/12051044/GoldenDict.crash.report.txt)

Since I had already downloaded a functioning version of the app from a different source (as mentioned in my reply to @shenlebantongying [above](https://github.com/xiaoyifang/goldendict-ng/discussions/962#discussioncomment-6432368)), I will just use that version for now -- though it didn't quite work out of the box either but only worked after some system tweaking (following the instruction here: ). I'm not too sure how the tweaking has affected my system or how trustworthy the developer really is, but at least I can normally use the app now.

**Reply by xiaoyifang** [reply]
https://github.com/xiaoyifang/goldendict-ng/issues/814

the same as the above issue,

you have used an old release ,try some package from here https://github.com/xiaoyifang/goldendict-ng/releases/tag/tag-alpha.d9bf1d1a
